src/datatablerow/datatablerow.jl

```diff
@@ -4,10 +4,6 @@ immutable DataTableRow{T <: AbstractDataTable}
     row::Int
 end
 
-function Base.getindex(r::DataTableRow, idx::AbstractArray)
```

Comment:
Technically, the second one has more generic signature, so it would be applied in `AbstractArray` case as well, although it would return a `DataTable`, not a `DataTableRow`. I don't recall if there was any need to delete the first method, but I suppose returning `DataTableRow` (i.e. keeping the first method) is a more consistent behaviour.